src/App.tsx

```diff
@@ -1,24 +1,348 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  createTodos,
+  getTodos,
+  removeTodo,
+  updateTodosCompleted,
+  updateTodosTitle,
+} from './api/todos';
+import { TodosList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoError } from './components/TodoError/TodoError';
+import { TodoStatus } from './types/TodoStatus';
 
-const USER_ID = 0;
+const statusTodos = (todos: Todo[], filterBy: TodoStatus) => {
+  let filteredTodos = todos;
+
+  switch (filterBy) {
+    case TodoStatus.Active:
+      filteredTodos = todos.filter(item => !item.completed);
+      break;
+    case TodoStatus.Completed:
+      filteredTodos = todos.filter(item => item.completed);
+      break;
+    case TodoStatus.All:
+    default:
+      break;
+  }
+
+  return filteredTodos;
+};
+
+const USER_ID = 6709;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoadingCompleted, setIsLoadingComoleted] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isEditing, setIsEditing] = useState(0);
+  const [isLoading, setIsLoading] = useState(0);
+  const [activeFilter, setActiveFilter] = useState<TodoStatus>(TodoStatus.All);
+  const [error, setError] = useState('');
+  const [deleteTodoId, setDeleteTodoId] = useState(0);
+  const notCompletedTodo = todos.filter(todo => !todo.completed).length;
+  const completedTodo = todos.filter(todo => todo.completed).length;
```

Comment:
```suggestion
  const completedTodosCount = todos.filter(todo => todo.completed).length;
```
I'd better specify that is the number of completed todos, not todos itself